common/containers/Tabs/SendTransaction/index.tsx

```diff
@@ -433,6 +442,50 @@ export class SendTransaction extends React.Component<Props, State> {
       )
     );
 
+  public generateWeb3TxFromState = async () => {
+    await this.resetJustTx();
+    const { nodeLib, wallet, gasPrice, network } = this.props;
+
+    const { token, unit, value, to, data, gasLimit } = this.state;
+    const chainId = network.chainId;
+    const transactionInput = {
+      token,
+      unit,
+      value,
+      to,
+      data
+    };
+    const bigGasLimit = new Big(gasLimit);
+
+    if (!(wallet instanceof Web3Wallet)) {
+      return;
+    }
+
+    try {
+      const txHash = await confirmAndSendWeb3Transaction(
+        wallet,
+        nodeLib,
+        gasPrice,
+        bigGasLimit,
+        chainId,
+        transactionInput
+      );
+
+      this.props.showNotification(
+        'success',
+        <TransactionSucceeded
+          txHash={txHash}
+          blockExplorer={network.blockExplorer}
```

Comment:
The props for this component error in VSCode, though Webpack compiles without throwing and things seem to run fine in the browser. AFAIK, it's the same setup in [the wallet saga](https://github.com/MyEtherWallet/MyEtherWallet/blob/5d4b36d453631a77513b6b55f5bc2732eeded97f/common/sagas/wallet.tsx#L165). Any idea what I'm missing, @HenryNguyen5 ?